homeassistant/components/onewire/const.py

```diff
@@ -20,6 +20,25 @@
 DEVICE_KEYS_0_7 = range(8)
 DEVICE_KEYS_A_B = ("A", "B")
 
+DEVICE_SUPPORT_OWSERVER = {
+    "05": (),
+    "10": (),
+    "12": (),
+    "1D": (),
+    "1F": (),
+    "22": (),
+    "26": (),
+    "28": (),
+    "29": (),
+    "3A": (),
+    "3B": (),
+    "42": (),
+    "7E": ("EDS0066", "EDS0068"),
+    "EF": ("HB_MOISTURE_METER", "HobbyBoards_EF"),
```

Comment:
If this goes in after mine, we will need to add HB_HUB

Reply:
Yes, which ever one is merged first, the other will have to be rebased.